.github/workflows/ci.yml

```diff
@@ -4,144 +4,88 @@ on:
   push:
     branches:
       - master
-    tags:
-      - v[0-9]+.[0-9]+.[0-9]+
+      - main
+      - "v*"
   pull_request:
 
 env:
   NODE_VERSION: 10
-  PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_number }}
-  PERCY_PARALLEL_TOTAL: 1
+  FORCE_COLOR: 1
 
 jobs:
   lint:
-    name: Lint files
+    name: Linting
     runs-on: ubuntu-latest
-    timeout-minutes: 3
-    steps:
-      - name: Check out a copy of the repo
-        uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ env.NODE_VERSION }}
-        uses: actions/setup-node@v2-beta
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Cache npm cache and node_modules
-        id: cache-dependencies
-        uses: actions/cache@v2
```

Comment:
I didn't understand why caching was removed. Can you provide explanation in the pull request description so that I have more context?

Reply:
again I said that I am using a template that was provided to me. I discussed this with a few colleagues and it is clear that adding caching like this doesn't really speed up the CI times all that much and **significantly** increases the cognitive load for anyone wanting to try and understand what is going on here. 

I can pull up some representative times to compare once the CI actually passes 😂